.github/workflows/test.yml

```diff
@@ -23,6 +23,11 @@ jobs:
   cypress-run:
     needs: iscodeclean
     runs-on: ubuntu-16.04
+    env:
+      CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+      CYPRESS_CACHE_FOLDER: "/home/runner/.cache/Cypress"
+      TERM: xterm
+      CI: 1
```

Comment:
why did you change this? It might be better i'm just not sure why it's better

Reply:
I think you should revert this because it seems to be causing issues by setting protected environment variables on github actions